### PR TITLE
Updated installation instructions

### PR DESCRIPTION
**Pull request by ThierryO**

### CODE REVIEW

File: content/installation/administrator/admin_install_rtools/index.md [hunk 1]
```diff
@@ -10,19 +10,19 @@ tags: ["r", "windows", "installation"]
 
 ### Installatie en upgrade
 
-De installatiebestanden zijn beschikbaar via https://cloud.r-project.org/bin/windows/Rtools/
+De installatiebestanden zijn beschikbaar via [cloud.r-project.org](https://cloud.r-project.org/bin/windows/Rtools/).
+Kies de 64-bit versie van de installer.
 
 Bij een upgrade dient eerst de vorige versie te worden verwijderd.
```
Comment (hansvancalster) [inline content/installation/administrator/admin_install_rtools/index.md:16]: This is not strictly necessary.  I have both installed. As long as the path variable points to the correct one.

File: content/installation/administrator/admin_install_r/index.Rmd [hunk 2]
```diff
@@ -19,57 +19,74 @@ knitr::opts_chunk$set(
...
+1. Kopieer het bestand [`Rprofile.site`](Rprofile.site) naar `etc` in de doelmap waar je R geïnstalleerd hebt (`C:\Program Files\R\R-3.x.y`). 
+Hierbij moet je het bestaande bestand overschrijven.
+1. Zorg dat de gebruiker schrijfrechten heeft voor `C:\R\R-4.x.y\library` en `C:\R\library`.
 
 #### Afwijkingen t.o.v. default installatie
 
+- Wijzig de standaarddoelmap naar `C:\R\R-4.x.y`
 - **alle** gebruikers moeten **volledige** rechten hebben in 
     - `C:\R\library`
     - `C:\Program Files\R\R-3.x.y\library`
```
[the remark this answers is not in the document]
Comment (hansvancalster) [reply content/installation/administrator/admin_install_r/index.Rmd:57]: Also update the md file if you commit this

**Comment by ThierryO** [comment]
@ElsLommelen Please post the differences with your `.gitconfig`. Maybe we can use that to improve the current `.gitconfig`.

**Comment by ElsLommelen** [comment]
I posted my `.gitconfig` in PR #186.

**Pull request merged**